src/App.js

```diff
@@ -20,11 +17,20 @@ class App extends React.Component {
   handleChange(e) {
     const data = calculate(this.state, e);
     this.setState(data);
+    console.log(data);
```

Comment:
- [ ] Please delete console.log to follow the Javascript best [practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log%20to%20your%20repo.%20It%E2%80%99s%20ugly%2C%20it%20kills%20performance%20and%20it%20can%20make%20confidential%20data%20be%20visible%20to%20anyone%20using%20the%20browser%20tools%20to%20look%20at%20your%20website.).